source/pool/contracts/Pool.sol

```diff
@@ -128,34 +131,55 @@ contract Pool is IPool, Ownable2Step {
   /**
    * @notice Set claims from previous pool contract
    * @dev Only owner
-   * @param _root bytes32
+   * @param _proposalId bytes32
    * @param _accounts address[]
    */
   function setClaimed(
-    bytes32 _root,
+    bytes32 _proposalId,
     address[] memory _accounts
   ) external override multiAdmin {
-    if (roots[_root] == false) {
-      roots[_root] = true;
+    if (roots[rootIds[_proposalId]] == false) {
+      roots[rootIds[_proposalId]] = true;
     }
     for (uint256 i = 0; i < _accounts.length; i++) {
       address account = _accounts[i];
-      if (claimed[_root][account]) revert AlreadyClaimed();
-      claimed[_root][account] = true;
+      if (claimed[rootIds[_proposalId]][account]) revert AlreadyClaimed();
+      claimed[rootIds[_proposalId]][account] = true;
     }
-    emit Enable(_root);
+    emit Enable(rootIds[_proposalId]);
   }
 
   /**
    * @notice Enables claims for a merkle tree of a set of scores
    * @param _root bytes32
    */
-  function enable(bytes32 _root) external override multiAdmin {
+  function enable(
```

Comment:
Really minor, but probably worth switching the order of these now, `enable(proposalId, root)` - "enable the proposal by adding this root".

Reply:
Done in the new commit